cmplibrary/src/main/res/raw/js_receiver.js

```diff
@@ -2,18 +2,25 @@ function isFromPM(payload) {
     return payload.fromPM || (payload.settings && payload.settings.vendorList != null);
 }
 
-function getPmIdFromURL(url) {
-    return url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
+function getPmIdAndPmTabFromURL(url) {
+    let pmId = url ? url.match(/[?&]message_id(=([^&#]*)|&|#|$)/)[2] : null;
+    let pmTabShow = null;
+    if(url &&  url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/) && url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/)[2]){
+       pmTabShow = url.match(/[?&]pmTab(=([^&#]*)|&|#|$)/)[2];
+     }
+    return [pmId, pmTabShow];
 }
 
 function actionFromMessage(payload) {
     var actionPayload = payload.actions && payload.actions.length && payload.actions[0] && payload.actions[0].data ? payload.actions[0].data : {};
+    let pmDetails = getPmIdAndPmTabFromURL(actionPayload.iframe_url);
     return {
         name: payload.name,
         actionType: actionPayload.type,
         choiceId: String(actionPayload.choice_id),
         requestFromPm: false,
-        pmId: getPmIdFromURL(actionPayload.iframe_url),
+        pmId: pmDetails[0],
```

Comment:
here we could use:
{
...
  pmId:pmDetails.pmId,
  pmTab:pmDetails.pmTab,
...
}

Reply:
Done